Brokerages/QuantConnect.Brokerages.csproj

```diff
@@ -270,21 +301,38 @@
     <Compile Include="Alpaca\Markets\Messages\JsonListenRequest.cs" />
     <Compile Include="Alpaca\Markets\Messages\JsonMinuteAgg.cs" />
     <Compile Include="Alpaca\Markets\Messages\JsonNewOrder.cs" />
+    <Compile Include="Alpaca\Markets\Messages\JsonNewOrderAdvancedAttributes.cs" />
     <Compile Include="Alpaca\Markets\Messages\JsonOrder.cs" />
+    <Compile Include="Alpaca\Markets\Messages\JsonOrderActionStatus.cs" />
+    <Compile Include="Alpaca\Markets\Messages\JsonPortfolioHistory.cs" />
     <Compile Include="Alpaca\Markets\Messages\JsonPosition.cs" />
+    <Compile Include="Alpaca\Markets\Messages\JsonPositionActionStatus.cs" />
     <Compile Include="Alpaca\Markets\Messages\JsonStreamAgg.cs" />
     <Compile Include="Alpaca\Markets\Messages\JsonStreamQuote.cs" />
     <Compile Include="Alpaca\Markets\Messages\JsonStreamTrade.cs" />
     <Compile Include="Alpaca\Markets\Messages\JsonTradeUpdate.cs" />
     <Compile Include="Alpaca\Markets\Messages\JsonUnsubscribeRequest.cs" />
+    <Compile Include="Alpaca\Markets\Parameters\AccountActivitiesRequest.cs" />
+    <Compile Include="Alpaca\Markets\Parameters\AggregatesRequest.cs" />
+    <Compile Include="Alpaca\Markets\Parameters\AggregationPeriod.cs" />
+    <Compile Include="Alpaca\Markets\Parameters\AlpacaTradingClientConfiguration.cs" />
+    <Compile Include="Alpaca\Markets\Parameters\AssetsRequest.cs" />
+    <Compile Include="Alpaca\Markets\Parameters\CalendarRequest.cs" />
+    <Compile Include="Alpaca\Markets\Parameters\ChangeOrderRequest.cs" />
+    <Compile Include="Alpaca\Markets\Parameters\HistoricalRequest.cs" />
+    <Compile Include="Alpaca\Markets\Parameters\HistoryPeriod.cs" />
+    <Compile Include="Alpaca\Markets\Parameters\ListOrdersRequest.cs" />
+    <Compile Include="Alpaca\Markets\Parameters\NewOrderRequest.cs" />
+    <Compile Include="Alpaca\Markets\Parameters\PolygonDataClientConfiguration.cs" />
     <Compile Include="Alpaca\Markets\Parameters\PolygonStreamingClientConfiguration.cs" />
+    <Compile Include="Alpaca\Markets\Parameters\PortfolioHistoryRequest.cs" />
     <Compile Include="Alpaca\Markets\Parameters\StreamingClientConfiguration.cs" />
+    <Compile Include="Alpaca\Markets\PolygonDataClient.cs" />
     <Compile Include="Alpaca\Markets\PolygonStreamingClient.cs" />
     <Compile Include="Alpaca\Markets\RateThrottler.cs" />
     <Compile Include="Alpaca\Markets\RestClient.cs" />
```

Comment:
I see a lot of new files added but only a couple removed, should we remove more, like for example `RestClient`?